sound/soc/codecs/tlv320aic32x4.c

```diff
@@ -1162,6 +1162,12 @@ int aic32x4_probe(struct device *dev, struct regmap *regmap)
 	if (IS_ERR(aic32x4->mclk)) {
 		dev_err(dev, "Failed getting the mclk. The current implementation does not support the usage of this codec without mclk\n");
 		return PTR_ERR(aic32x4->mclk);
+	} else {
```

Comment:
I suspect you'll get the comment from upstream that the else and extra braces aren't required as you've ```return```ed within the ```if```.
Just add:
```
	ret = clk_prepare_enable(aic32x4->mclk);
	if (ret != 0) {
		dev_err(dev, "Failed to enable MCLK: %d\n", ret);
		return ret;
	}
```